subsys/bluetooth/host/id.c

```diff
@@ -1345,7 +1348,7 @@ static inline bool rpa_timeout_valid_check(void)
 {
 #if defined(CONFIG_BT_PRIVACY)
 	/* Check if create conn timeout will happen before RPA timeout. */
-	return k_delayed_work_remaining_get(&bt_dev.rpa_update) >
+	return k_work_delayable_remaining_get(&bt_dev.rpa_update) >
```

Comment:
This looks wrong in units: you're comparing the return value in ticks to a multiple of 10 ms.

Please check *all* uses of `k_work_delayable_remaining_get` throughout the PR.

Reply:
Thanks, missed that this was returning ticks now.